src/snowflake/snowpark/modin/pandas/dataframe.py

```diff
@@ -1348,27 +1348,18 @@ def iterrow_builder(s):
         partition_iterator = SnowparkPandasRowPartitionIterator(self, iterrow_builder)
         yield from partition_iterator
 
-    @dataframe_not_implemented()
     def items(self):  # noqa: D200
         """
         Iterate over (column name, ``Series``) pairs.
         """
-        # TODO: SNOW-1063346: Modin upgrade - modin.pandas.DataFrame functions
+
         def items_builder(s):
             """Return tuple of the given `s` parameter name and the parameter themselves."""
             return s.name, s
 
         partition_iterator = PartitionIterator(self, 1, items_builder)
         yield from partition_iterator
 
-    @dataframe_not_implemented()
-    def iteritems(self):  # noqa: RT01, D200
```

Comment:
pandas 2.0 [removed](https://pandas.pydata.org/docs/whatsnew/v2.0.0.html#removal-of-prior-version-deprecations-changes) `iteritems`, an alias for `items`, in pandas 2.0, so no need to include it here.